app/Models/User.php

```diff
@@ -108,4 +109,13 @@ public function name(): string
 	{
 		return ($this->id == 0) ? 'Admin' : $this->username;
 	}
+
+	public function fullname(): string
```

Comment:
Accessor functions in Laravel use the somewhat acquired syntay `getFullnameAttribute`, see https://laravel.com/docs/8.x/eloquent-mutators#defining-an-accessor.

Reply:
luchee is using a different syntax. (See app/Models/User.php)
I think the lychee syntax is better.